### DISCUSSION TITLE
Placing the Button content in Center

### DISCUSSION THREAD

**Discussion by mayankjoshii** [hecrj/iced#1667] [General]
Hi all,

I have a UI code, that contains button and by default the text/label of button in placed in the left. I want the button text/label to placed in the center. Can anyone help me with the exact setting for button widget. 

here's my code sample for button. 
```
let installButton = button("INSTALL")
            .padding(10)
            .width(iced::Length::Units(110))
            .on_press(Message::ButtonPressed);
            
```

**Comment by mayankjoshii** [comment]
You can do something like this: 

```
let installButton = button(text("INSTALL").horizontal_alignment(alignment::Horizontal::Center))
            .padding(10)
            .width(iced::Length::Units(110))
            .on_press(Message::installButtonPressed);
```

**Comment by chefexperte** [comment]
Now that `horizontal_alignment` is not a thing anymore, how can this be done? `align_x` does not do anything with my button text :(

**Comment by vfosnar** [comment]
You can
```rust
button(
    Container::new(text("whatever"))
        .width(Length::Fill)
        .align_x(Alignment::Center)
)
.width(Length::Fill)
```
https://jl710.github.io/iced-guide/layout/4_container.html
